tests/platform_tests/test_intf_fec.py

```diff
@@ -130,15 +130,20 @@ def test_verify_fec_stats_counters(duthosts, enum_rand_one_per_hwsku_frontend_ho
         fec_symbol_err = intf.get('fec_symbol_err', '').lower()
         # Check if fec_corr, fec_uncorr, and fec_symbol_err are valid integers
         try:
-            int(fec_corr)
+            fec_corr_int = int(fec_corr)
             fec_uncorr_int = int(fec_uncorr)
-            int(fec_symbol_err)
+            fec_symbol_err_int = int(fec_symbol_err)
         except ValueError:
             pytest.fail("FEC stat counters are not valid integers for interface {}, \
                         fec_corr: {} fec_uncorr: {} fec_symbol_err: {}"
                         .format(intf_name, fec_corr, fec_uncorr, fec_symbol_err))
 
-        # Check for uncorrectable FEC errors
+        # Check for non-zero FEC uncorrectable errors
         if fec_uncorr_int > 0:
             pytest.fail("FEC uncorrectable errors are non-zero for interface {}: {}"
                         .format(intf_name, fec_uncorr_int))
+
+        # Check for valid FEC correctable codeword errors > FEC symbol errors
+        if fec_symbol_err_int > fec_corr_int:
```

Comment:
Can we make this >= since equal is also a valid case?

Reply:
@dgsudharsan sorry, I am not sure I got you completely, we are failing the test only when FEC_symbol_errors are higher, which makes equal as valid case.